js/src/Ice/UserException.js

```diff
@@ -0,0 +1,78 @@
+//
+// Copyright (c) ZeroC, Inc. All rights reserved.
+//
+
+import { Exception } from "./Exception.js";
+
+export class UserException extends Exception {
+    constructor() {
+        super();
+        Exception.captureStackTrace(this);
```

Comment:
We can probably remove this, not sure why we were capturing stack traces for user exceptions

Reply:
We shoud remove it. Can we then remove the constructor entirely?